contracts/MultiSigStub.sol

```diff
@@ -0,0 +1,67 @@
+pragma solidity ^0.4.15;
+
+/**
+ * @title MultiSigStub  
+ * Contact that delegates calls to a library to build a full MultiSigWallet that is cheap to create. 
```

Comment:
1) Typo (should be `Contact`). 

2) Perhaps should also mention which library the calls are delegated to (looks like it is a library contract expected to be deployed to address `0x370A93cd1DC15875fF02aa0b952D44Bb3dD905E5`, where is the .sol code for that ?)